engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/NadelRenameArgumentInputTypesTransform.kt

```diff
@@ -93,7 +93,10 @@ internal class NadelRenameArgumentInputTypesTransform : NadelTransform<State> {
             // reasons we don't do it.  Also, Nadel currently does not allow an input field to be renamed
             // and hence we don't have to change inner map keys either.
             //
-            return NormalizedInputValue(underlyingTypeName, inputValue.value)
+            val overallWrappedTypename = inputValue.typeName
+            val newUnderlyingTypeNameWithOriginalWrapping =
+                overallWrappedTypename.replace(overallTypeName, underlyingTypeName)
```

Comment:
hacky way to rename and preserve the original type's wrapping. let me know if there is a better way

Reply:
Ahh I feel like we should have a better way for this.  I looked in graphql java and it does in the AST Type sense but not from strings

`graphql.schema.idl.TypeUtil` say but thats AST element

Reply:
Maybe make a new util method somewhere so at least its common and can be improved

It can then be unit tested

`val newUnderlyingTypeNameWithOriginalWrapping =                FieldUtil.replaceTypeName(inputValue.typeName, underlyingTypeName)`